ansible/roles/nameindex/tasks/main.yml

```diff
@@ -23,8 +23,15 @@
   tags:
     - "nameindex"  
 
+- name: set name index to use (Custom)
+  set_fact:
+    lucene_namematching_url: "{{ custom_namematching_url }}"
```

Comment:
custom_namematching_url is a value that can be used to specify another lucene index to use. Im using the ALS and assume others will use for other deployments